JustTweak/Classes/UI/TweaksConfigurationViewController.swift

```diff
@@ -198,9 +212,10 @@ extension TweaksConfigurationViewController: TweaksConfigurationViewControllerCe
     
     internal func tweaksConfigurationCellDidChangeValue(_ cell: TweaksConfigurationViewControllerCell) {
         if let indexPath = tableView.indexPath(for: cell as! UITableViewCell) {
-            if let tweak = tweakAt(indexPath: indexPath) {
+            if var tweak = tweakAt(indexPath: indexPath) {
                 let configuration = configurationsCoordinator?.topCustomizableConfiguration()
                 configuration?.set(value: cell.value, forTweakWithIdentifier: tweak.identifier)
+                tweak.value = cell.value
```

Comment:
I used a Tweak class internal to the current class as the Tweak class internal to the framework is immutable.
Building the sections with `rebuildSections` using this class allows setting the value when changed by the user.